tests/prep_test_cli_rss.sh

```diff
@@ -19,7 +19,7 @@ fi
 
 export VASP_COMMAND=vasp.serial
 export VASP_COMMAND_GAMMA=vasp.gamma_serial
-export VASP_PP_PATH=${VASP_PATH}/pot/rev_54/PBE
```

Comment:
Why did you change this variable? It's internal to ASE's vasp calculator, and this isn't even a test - it's a script that does the reference gap_rss_iter_fit calculation which the corresponding test compares to.

Reply:
Do we need a more general discussion of how the VASP tests are configured?

Reply:
I first thought that was part of the reason why the pytest for VASP was skipped.
So this change make no sense after since I now that the pytest is not done in the pipeline.